simdev/util/pipeline.py

```diff
@@ -44,6 +48,27 @@ def run(self):
     def append(self, stage):
         self.stages.append(stage)
 
+    @staticmethod
+    def store_context(value, name):
+        file_path = os.path.join(SAVED_DIRECTORY, *name.split('/'))
```

Comment:
The whole purpose of this method is to store intermediate results (e.g. results from a specific stage) so that we can use them instead of recomputing the whole stage